mysql-test/suite/rocksdb/t/rocksdb_fault_injection-master.opt

```diff
@@ -1 +1,3 @@
---rocksdb_fault_injection_options="{"retry":true,"failure_ratio":1,"filetypes":["kTableFile"]}"
+# Specify DDSE as INNODB, since during DDSE upgrade system table,
+# myrocks drop index thread will crash due to fault inject IO error
+--rocksdb_fault_injection_options="{"retry":true,"failure_ratio":1,"filetypes":["kTableFile"]}" --default-dd-system-storage-engine=InnoDB
```

Comment:
This needs `--initialize` too in order to have the described effect (otherwise
MTR re-uses the default test instance, which gets changed on the test startup)

Reply:
If I had added --initialize, the instance will fail to initialize rocksdb plugin due to fault injection.

Reply:
Will that happen even if DDSE is InnoDB?